inc/saitam.h

```diff
@@ -1560,17 +1560,17 @@ typedef void (*sai_tam_event_notification_fn)(
  * @brief TAM telemetry data get API
  *
  * @param[in] switch_id SAI Switch object id
- * @param[in] attr_count Number of attributes
- * @param[in] attr_list Array of attributes
+ * @param[in] obj_list SAI Switch object list
```

Comment:
This API is part of sai v1.4, there are compatibility implications with that version if the signature is changed.  Could you please check.

Reply:
Stale Changes